packages/amplify-category-function/src/provider-utils/awscloudformation/utils/convertLayersTypes.ts

```diff
@@ -39,7 +39,7 @@ export const convertProjectLayersToExternalLayers = (lambdaLayers: LambdaLayer[]
 export const convertExternalLayersToProjectLayers = (lambdaLayers: LambdaLayer[], envName: string): LambdaLayer[] => {
   const modifiedLambdaLayers: LambdaLayer[] = [];
   lambdaLayers.forEach(layer => {
-    if (layer.type === externalLayer && layer.arn.hasOwnProperty(LAYER_ARN_KEY)) {
+    if (layer.type === externalLayer && Object.prototype.hasOwnProperty.call(layer.arn, LAYER_ARN_KEY)) {
```

Comment:
https://eslint.org/docs/latest/rules/no-prototype-builtins.
It is not _identical_, but it in all the situations in our code it is functionally the same. I would err with just complying with the lint rule here, because is a small fix, and it is rather uncommon to use the object prototype methods in our codebase anyways, so it shouldn't be overly onerous for developers to comply with going forward.